tests/unit/backend/corpora/common/entities/test_dataset.py

```diff
@@ -113,9 +112,9 @@ def test__create_ids__ok(self):
         )
 
         dataset_id = dataset.id
-        artifact_ids = get_ids(dataset.artifacts)
-        deployment_directory_ids = get_ids(dataset.deployment_directories)
-        contributor_ids = get_ids(dataset.contributors)
+        artifact_ids = [i.id for i in dataset.artifacts]
+        deployment_directory_ids = [i.id for i in dataset.deployment_directories]
+        contributor_ids = [i.id for i in dataset.contributors]
```

Comment:
Are these variables used before `dataset` is redefined on line 122?

Reply:
Oops! They are. Resolving.

Reply:
However, I will point out this as an example for clearer variable names (distinguishing "created dataset" values and "retrieved from DB dataset" values)

Reply:
fixed